Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:2.7 AS no-spark
 
```

Comment:
Since now we are using `stretch-backports`, suggest to use `python:2.7-stretch` to be make the intent explicit (currently `python:2.7` and `python:2.7-stretch` are the same image). Also helps to convey what the underlying distribution is.

Reply:
ok sure